benten-core/src/main/java/com/intuit/benten/nlp/dialogflow/DialogFlowClient.java

```diff
@@ -22,57 +17,48 @@
-        configuration =
-                new AIConfiguration(token);
-        if (bentenProxyConfig.isProxyEnabled()) {
-            SocketAddress socketAddress =
-                    new InetSocketAddress(bentenProxyConfig.getHost(), bentenProxyConfig.getPort());
-            Proxy proxy =
-                    new Proxy(Proxy.Type.valueOf(bentenProxyConfig.getProtocol().toUpperCase()), socketAddress);
-            configuration.setProxy(proxy);
-        }
-        aiDataService =
-                new AIDataService(configuration);
+    public DialogFlowClient(String PROJECT_ID) {
+        this.PROJECT_ID = PROJECT_ID;
     }
 
-
     @Override
-    public BentenMessage sendText(String text, String sessionId) throws AiException {
+    public BentenMessage sendText(String text, String sessionId) {
         return sendText(text,sessionId,false);
     }
 
     @Override
-    public BentenMessage sendText(String text, String sessionId, boolean reset) throws AiException {
-        AIServiceContext aiServiceContext =
-                AIServiceContextBuilder.buildFromSessionId(sessionId);
-
-        AIRequest request =
-                new AIRequest(text);
+    public BentenMessage sendText(String text, String sessionId, boolean reset) {
 
-        AIResponse response;
-        BentenMessage bentenMessage;
+        SessionsClient sessionsClient = null;
         try {
-            if(reset){
-                aiDataService.resetActiveContexts(aiServiceContext);
-            }
-
-            response = aiDataService.request(request, aiServiceContext);
+            sessionsClient = SessionsClient.create();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        SessionName session = SessionName.of(PROJECT_ID, sessionId);
+            BentenMessage bentenMessage;
+
+            Builder textInput = TextInput.newBuilder().setText(text).setLanguageCode("en-US");
+            // Build the query with the TextInput
+            QueryInput queryInput = QueryInput.newBuilder().setText(textInput).build();
+            DetectIntentRequest detectIntentRequest =
+                    DetectIntentRequest.newBuilder()
+                        .setSession(session.toString())
+                        .setQueryInput(queryInput)
+                        .build();
+            DetectIntentResponse response = sessionsClient.detectIntent(detectIntentRequest);
+            // Display the query result
+            QueryResult queryResult = response.getQueryResult();
 
             bentenMessage = new BentenMessage();
-            bentenMessage.setSpeech(response.getResult().getFulfillment().getSpeech());
-            bentenMessage.setAction(response.getResult().getAction());
-            bentenMessage.setIsActionComplete(!response.getResult().isActionIncomplete());
-            bentenMessage.setParameters(response.getResult().getParameters());
-
-        } catch (Exception e) {
```

Comment:
This exception handling should not be removed. Can you add this back ?

Reply:
Added the handling back